src/components/TodosItem/TodosItem.tsx

```diff
@@ -0,0 +1,107 @@
+import classNames from 'classnames';
+import { useContext, useState } from 'react';
+import { Todo } from '../../types/Todo';
+import { Error } from '../../types/Error';
+import { deleteTodo, patchTodo } from '../../api/todos';
+import { TodosContext } from '../TodosContext';
+import { AuthContext } from '../Auth/AuthContext';
+
+type Props = {
+  todo: Todo,
+};
+
+export const TodosItem: React.FC<Props> = ({ todo }) => {
+  const [query, setQuery] = useState('');
+  const [isLoad, setisLoad] = useState(false);
+
+  const providerValue = useContext(TodosContext);
```

Comment:
That's nice you created context to optimize your app, but it seems redundant here. Because you pass almost all things from `providerValue` to `TodoList` and other components but use it only here
So I guess better to pass values you need from `TodoList`